lib/agent0/agent0/logging/rollbar_utilities.py

```diff
@@ -0,0 +1,34 @@
+"""Utilities for rollbar."""
+import os
+
+import rollbar
+from dotenv import load_dotenv
+
+ROLLBAR_API_KEY = os.getenv("ROLLBAR_API_KEY")
+
+
+def initialize_rollbar(environment_name: str) -> bool:
+    """Initializes the rollbar sdk.
+
+    Parameters
+    ----------
+    environment_name : str
+        The name of the environment.  Should be something like 'local.fuzzbots' or 'aws.fuzzbots'
+
+    Returns
+    -------
+    bool
+        True if rollbar is initialized.
+    """
+
+    load_dotenv("rollbar.env")
+    log_to_rollbar = bool(ROLLBAR_API_KEY)
+    if log_to_rollbar:
+        print("logging to rollbar enabled.")
```

Comment:
I think this should be logged, e.g. `logging.info("logging to rollbar enabled.")`